changelog-extractor.py

```diff
@@ -0,0 +1,144 @@
+"""Script parses changelog from git based on 2 latest tags, formates changelog to .md format, creates/updates release on gitlab, posts to slack
+"""
+import subprocess
+import os
+
+def getMdFormattedChangelog(newCommit, oldCommit) : 
```

Comment:
methods should be in snake casing, not in camel casing, according to Python conventions. But is ok for them to be like this, since all the methods are consistent with each other

Reply:
snakes-cased